src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -1082,63 +1079,51 @@ internal static string GetProcessShortName(String name)
 
         // native struct defined in ntexapi.h
         [StructLayout(LayoutKind.Sequential)]
-        internal class SystemProcessInformation
+        internal unsafe struct SystemProcessInformation
         {
             internal uint NextEntryOffset;
             internal uint NumberOfThreads;
-            private long _SpareLi1;
-            private long _SpareLi2;
-            private long _SpareLi3;
-            private long _CreateTime;
-            private long _UserTime;
-            private long _KernelTime;
+            private fixed byte Reserved1[48];
 
             internal ushort NameLength;   // UNICODE_STRING   
```

Comment:
I'd prefer it if we actually defined a UNICODE_STRING struct which is then used here as ImageName.